src/source/publishing/influx_publisher.cr

```diff
@@ -36,48 +55,124 @@ module PlaceOS::Source
       end
     end
 
+    NO_POINTS = [] of Flux::Point
```

Comment:
could returning this reference to an empty array be risky in terms of potential mutation of the returned array?